test/controllers/APIControllerSpec.scala

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import org.scalatestplus.play._
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.test.Helpers.contentAsString
+import play.api.test.{FakeRequest, Injecting}
+
+import scala.concurrent.Future
+import play.api.mvc._
+import play.api.test.Helpers._
+
+import java.time.{LocalDate}
+
+class APIControllerSpec
+    extends PlaySpec
+    with Results
+    with GuiceOneAppPerTest
+    with Injecting {
+
+  "APIController" should {
```

Comment:
Each "suite" is intended to test a particular set of functionality. in this case, you are mixing "status" endpoint tests with "findAllProduct" endpoint tests. If you plan to do that, you'll need to be more specific in the naming of the tests